.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
+/workspace.xml
```

Comment:
We should probably setup .gitignore properly at base dir.

Reply:
Done. Copied from Presto